### ISSUE TITLE
Consolidate grammar to decrease complexity for other UpSet implementations

### ISSUE THREAD

**Issue by JakeWags** [visdesignlab/upset2#300]
Alt Text:
> "We are accessing `rawData["sets"]`, particularly for each set in `rawData["sets"]`, we are accessing `["elementName"]`, `rawData["sets"]["size"]`, then `rawData["items"]` and `rawData["setColumns"]`"

The grammar should be consolidated so that the alt text generator can remove any usage of `rawData`. This is because many (if not all) other UpSet implementations will not have the same data access as multinet provides for UpSet 2.

In doing this, adding a grammar and alt-text generator to other versions of UpSet will be simpler and easier for authors/contributors.

**Comment by JakeWags** [comment]
tasks:
- [ ] add "visible" flag to `processedData`
